theano/gpuarray/dnn.py

```diff
@@ -2800,6 +2800,204 @@ def local_abstractconv3d_cudnn_graph(op, context_name, inputs, outputs):
     return [rval]
 
 
+class _GpuDnnTransformerDescriptor(COp):
```

Comment:
Remove the underscore at top of name: `_GpuDnnTransformerDescriptor` -> `GpuDnnTransformerDescriptor`.